### ISSUE TITLE
azure-webapp-maven-plugin scales App Service plan (down) on deployment

### ISSUE THREAD

**Issue by viters** [microsoft/azure-maven-plugins#908]
### Plugin name and version
azure-webapp-maven-plugin 1.8.0

### Plugin configuration in your `pom.xml`
```xml
<plugin>
  <groupId>com.microsoft.azure</groupId>
  <artifactId>azure-webapp-maven-plugin</artifactId>
  <!-- check Maven Central for the latest version -->
  <version>${azure-webapp-maven-plugin.version}</version>
  <configuration>
    <schemaVersion>v2</schemaVersion>
    <resourceGroup>${azure.resource.group}</resourceGroup>
    <appName>${azure.app.name}</appName>
    <runtime>
      <os>linux</os>
      <javaVersion>jre8</javaVersion>
      <webContainer>tomcat 8.5</webContainer>
    </runtime>
    <deployment>
      <resources>
        <resource>
          <directory>${project.basedir}/target</directory>
          <includes>
            <include>*.war</include>
          </includes>
        </resource>
      </resources>
    </deployment>
  </configuration>
</plugin>
```

### Expected behavior
I have an existing Web App with existing App Service Plan. Plugin is configured to deploy to this specific app. This Web App and App Service Plan is configured. App Service Plan is P2V2. **Deployment with plugin should not change existing plans until explicitly configured**.

### Actual behavior
After deployment my existing App Service Plan is scaled down to P1V2. This process makes my other apps crash because of insufficient RAM.
**It bothers me that I have to configure each project and make sure all projects have correct pricingTier. Moreover, when I want to scale my plan I also have to update codebase of all projects using this plan.**

### Steps to reproduce the problem
Create an App Service Plan with P2V2. Create a Web App connected with this plan. Deploy to this plan with maven job with provided configuration. Check App Service Plan scaling.

**Comment by Flanker32** [comment]
@viters Thanks for your report and sorry for the inconvenience, we will fix this issue soon.

**Comment by Flanker32** [comment]
Fixed in #911
